src/SFA.DAS.ProviderCommitments/SFA.DAS.Commitments.Shared/Services/CommitmentsService.cs

```diff
@@ -1,15 +1,14 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
 using SFA.DAS.Commitments.Shared.Interfaces;
 using SFA.DAS.Commitments.Shared.Models;
 using SFA.DAS.CommitmentsV2.Api.Client;
 using SFA.DAS.CommitmentsV2.Api.Types.Requests;
 using SFA.DAS.CommitmentsV2.Api.Types.Responses;
-using SFA.DAS.CommitmentsV2.Types;
 using SFA.DAS.Encoding;
 
+
 namespace SFA.DAS.Commitments.Shared.Services
 {
     public class CommitmentsService : ICommitmentsService
```

Comment:
`CommtmentsService` is completely redundant and we're taking things out of it slowly but surely. It adds no value - we should call the api directly.

Reply:
If we get rid of the service then the controller will end up directly calling the API. If this is the case where would you place any business logic? It fits into the comment below as we could place the creation of the view model or the filter model into the service instead potentially if its only going to be called by the controller.

Reply:
I've moved the service code into a mapper as requested and removed the service calls